packages/cli/blueprints/ember-cli-build.js

```diff
@@ -0,0 +1,63 @@
+/*eslint-disable node/no-missing-require */
```

Comment:
I couldn't get this to pass even when adding the embroider packages as deps in the cli. Let me know if you have any other ideas.

Reply:
It's checking for the actual installed packages in node_modules, which won't be present yet since we don't run `yarn install` until after copying the blueprint elsewhere.

We could probably disable `no-missing-require` for the whole blueprints directory. That still gives us the `no-extraneous-require` rule, which is nearly as good and only compares with the contents of package.json.